src/main/java/eu/europeana/metis/sandbox/config/amqp/CreatedConfiguration.java

```diff
@@ -21,6 +21,9 @@
   @Value("${sandbox.rabbitmq.queues.record.created.dlq}")
   private String dlq;
 
+  @Value("${sandbox.rabbitmq.exchange.dlq}")
+  private String exchangeDlq;
+
   private AmqpConfiguration amqpConfiguration;
 
   public CreatedConfiguration(
```

Comment:
There is a high degree of similarity between all configurations in this package. Is there anything you can do to generalize?

Reply:
I found [this](https://docs.spring.io/spring-amqp/reference/html/#collection-declaration) I think is the best option available, let me know what do you think so I can look in more depth

Reply:
I will explore this in the next review, so that we can close this one and I can merge it to the other one I have pending if that is fine with you